api/envoy/config/filter/http/jwt_authn/v2alpha/config.proto

```diff
@@ -30,7 +31,30 @@ import "validate/validate.proto";
 //       - seconds: 300
 //
 // [#not-implemented-hide:]
-message JwtRule {
+message JwtProvider {
+  // A unique ID to uniquely identify a provider. It can be the same as "issuer".
+  // If an issuer have different audiences or other configurations, multiple JwtProvider
+  // can be used with different ids.
+  //
+  // Example of two providers of different audiences with the same issuer.
+  //
+  // .. code-block:: yaml
+  //
+  //     id: example.com-bookstore-audience
+  //     issuer: https://example.com
+  //     local_jwks:
+  //      - inline_string: "ACADADADADA"
+  //     audiences:
+  //     - bookstore
+  //
+  //     id: example.com-any-audience
+  //     issuer: https://example.com
+  //     local_jwks:
+  //      - inline_string: "ACADADADADA"
+  //
+  // Use 9 as the proto index number to support backward compatibility.
+  string id = 9 [(validate.rules).string.min_bytes = 1];
```

Comment:
perhaps `name` is more consistent across envoy configs?

Reply:
Done